applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -250,6 +259,7 @@ static bool app_event_handler(const struct app_event_header *aeh)
 				k_work_init_delayable(&dvfs_state_timeouts[i].timeout_work,
 						      dvfs_state_timeout_work_handler);
 			}
+			process_dvfs_states(DVFS_STATE_INITIALIZING, true);
```

Comment:
This needs to be done conditionally (keep in mind user could disable this feature in configuration now)